src/main/kotlin/com/github/avrokotlin/avro4k/schema/ClassSchemaFor.kt

```diff
@@ -28,7 +28,7 @@ class ClassSchemaFor(
 ) : SchemaFor {
 
    private val entityAnnotations = AnnotationExtractor(descriptor.annotations)
-   private val naming = RecordNaming(descriptor)
+   private val naming = RecordNaming(descriptor, DefaultNamingStrategy)
```

Comment:
Wouldn't the namingStrategy be needed here? Otherwise, the name of the Schema wouldn't be changed according to the strategy.

Reply:
This also probably is a field that should not run through the naming strategy. Or same thing as with enums but now with the example it generates something like this for Interface record -> 

```
{ 
 "type" : "record",
  "name" : "_interface",
  "namespace" : "com.github.avrokotlin.avro4k.schema.NamingStrategySchemaTest",
 ...
```